src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -115,6 +127,7 @@ private bool ShouldIncludeInTargetTypedCompletionList(ISymbol symbol, ImmutableA
                                select g;
 
             var itemListBuilder = ImmutableArray.CreateBuilder<CompletionItem>();
+            var typeConvertibilityCache = new Dictionary<ITypeSymbol, bool>();
```

Comment:
Should this just be ignoring nullability in general? Put another way, if one symbol is a `IEnumerable<string?>` and the other is an `IEnumerable<string>`, the convertibility is the same, right? If so, stick a AllNullabilityIgnoringSymbolComparer.Instance in as your key comparer.

Warning: it's not actually implemented to ignore nested nullability, but once the compiler fixes that then this will do the right thing. It *will* ignore top-level nullability.